CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to Nexus Operator
+
+## Have found a bug or have a feature request?
+
+Please, open an issue for us. That will really help us to improve and also others that are using our project. There are templates ready for bug reporting and feature request to make things easier for you.
+
+## Have any questions?
+
+We're happy to answer! Either [open an issue](https://github.com/m88i/nexus-operator/issues) or send an email to our mailing list: [nexus-operator@googlegroups.com](mailto:nexus-operator@googlegroups.com).
+
+## Are willing to send a PR?
+
+Before sending a PR, consider opening an issue first. So we can discuss your approach and the motivations behind it.
+
+Regarding your local development environment:
+
+1. We use [golint-ci](https://golangci-lint.run/) to check the code. Consider [integrating it in your favorite IDE](https://golangci-lint.run/usage/integrations/) to avoid failing in the CI
+2. **Always** run `make test` before sending a PR to make sure the license headers and the manifests are updated (and of course the unit tests are passing)
+3. Consider adding a new [end-to-end](https://sdk.operatorframework.io/docs/golang/e2e-tests/) test case covering your scenario and make sure to run `make test-e2e` before sending the PR
```

Comment:
```
4. Make sure to always keep your version of `Go` and the `operator-sdk` on par with the project. The current version information may be found at [our README pre-requisites section](https://github.com/m88i/nexus-operator#pre-requisites)
```

Wdyt? We still don't have that info on our pre-requisites section though. If you agree with this suggestion, would you mind putting that on the README as well?

Reply:
It's worth mentioning here. I'll update the doc.